src/test/kotlin/com/cultureamp/common/ClassReflectionTest.kt

```diff
@@ -166,3 +174,7 @@ sealed class FSubSubSubSubSub : FSubSubSubSub()
 data class F6(val value: String) : FSubSubSubSubSub()
 sealed class FSubSubSubSubSubSub : FSubSubSubSubSub()
 data class F7(val value: String) : FSubSubSubSubSubSub()
+
+sealed interface G
+data class G1(val value: String) : G
+data class G2(val value: Int) : G
```

Comment:
reading up on this it seems you used to be able to only use sealed subclasses if they were in the same file. worth putting them into separate files just to be sure?

Reply:
True - let's do it as another PR though before I lose track of this :P